plugins/stronghold/Cargo.toml

```
@@ -21,11 +21,11 @@ serde_json = { workspace = true }
 tauri = { workspace = true }
 log = { workspace = true }
 thiserror = { workspace = true }
-iota_stronghold = "1"
+iota_stronghold = "2.0.0"
```

```suggestion
iota_stronghold = "2"
```
just to make renovate less annoying lol